frontend/src/components/style/Buttons.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components'
+
+const StyledButton = styled.button`
+    cursor: pointer;
+    color: white;
+    font-size: 1em;
+    padding: 0.25em 1em;
+    border: 2px solid #1890ff;
+    border-radius: 5px;
+    background-color: #1890ff;
+    transition: all 0.3s;
+
+    :hover {
```

Comment:
Does hover styling work like that? I've always used '&:hover'.

Reply:
Yes, it does. I tested it.

Reply:
Cool, LGTM.